### PR TITLE
Moving files from old repo

### PR DESCRIPTION
**Pull request by darkwing**
\o/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "loads-template-v3",
```
Comment (pdehaan) [inline package.json]: "loads-web"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "description": "Testing Hapi w/ Angular routing",
+  "version": "1.0.0",
+  "author": "Peter deHaan <peter@deseloper.com> (http://nodeexamples.com/)",
```
Comment (pdehaan) [inline package.json]: "Mozilla (https://mozilla.org)"

File: package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "bugs": "https://github.com/pdehaan/loads-template-v3/issues",
```
Comment (pdehaan) [inline package.json]: OK, basically audit the whole file.
`homepage`
`license` (MPL)
`repository`
whatever else you think.

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "angular-routes",
+  "version": "1.0.0",
+  "authors": [
+    "Peter deHaan <peter@deseloper.com>"
```
Comment (pdehaan) [inline bower.json]: "Mozilla (https://mozilla.org)"

Actually, probably copy whatever settings make sense from the updated package.json.

**Comment by pdehaan** [comment]
Apart from tweaking the bower.json and package.json files, :+1: from me.
Or ignore me, merge it, and I can do a follow on PR.

**Comment by darkwing** [comment]
Updated!

**Comment by darkwing** [comment]
Going to merge so I can keep annihilating.

**Pull request merged**